config/vufind/facets.ini

```diff
@@ -134,6 +134,12 @@ facet_limit = 30
 ; Override facet_limit on a per-field basis using this array:
 ;facet_limit_by_field[format] = 50
 
+; Prefix a facet on a per-field basis:
```

Comment:
Might help to expand this comment to something like "Limit facets based on a prefix on a per-field basis:"